lib/Target/SubstraitPB/Export.cpp

```diff
@@ -168,13 +170,15 @@ void SubstraitExporter::exportAdvancedExtension(ExtensibleOpInterface op,
 
   // Set `optimization` field if present.
   if (optimizationAttr) {
-    std::unique_ptr<pb::Any> optimization = exportAny(optimizationAttr);
+    std::unique_ptr<::google::protobuf::Any> optimization =
```

Comment:
Hmm... this is another disadvantage of not using anonymous namespaces, which I didn't see mentioned in the resources I have linked to. Not being able to use namespace alias does make some code more verbose. Putting the alias outside of the namespace also has it problems (though I don't remember exactly what it was).